inst/local_tex/tex/latex/inbogeneric_2015/inbocolours_2015.sty

```diff
@@ -0,0 +1,23 @@
+%
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesPackage{inbocolours_2015}
+
+% definition of INBO colours
+\RequirePackage{xcolor}
+\definecolor{main.colour}{RGB}{192, 67, 132}
+\definecolor{sectiontitle.colour}{RGB}{255, 255, 255}
+\definecolor{frametitle.colour}{RGB}{192, 67, 132}
+\definecolor{support.colour}{RGB}{192, 67, 132}
+\definecolor{box.colour.title}{RGB}{255, 255, 255}
+\definecolor{box.colour.body}{RGB}{66, 142, 236}
+\definecolor{box.colour}{RGB}{53, 97, 150}
+\definecolor{alert.colour.title}{RGB}{255, 255, 255}
+\definecolor{alert.colour.body}{RGB}{228, 56, 145}
+\definecolor{alert.colour}{RGB}{132, 56, 96}
+\definecolor{example.colour.title}{RGB}{255, 255, 255}
+\definecolor{example.colour.body}{RGB}{68, 196, 110}
+\definecolor{example.colour}{RGB}{45, 125, 71}
+\definecolor{warning.colour}{RGB}{192, 67, 132}
+\definecolor{highlight.colour}{RGB}{192, 67, 132}
+\definecolor{link.colour}{RGB}{53, 97, 150}
+\definecolor{cite.colour}{RGB}{58, 163, 92}
```

Comment:
@ThierryO Shouldn't we copy the values of `highlight.colour` to `link.colour` and `cite.colour`?